src/main/java/com/woowacamp/soolsool/core/liquor/dto/response/PageLiquorClickResponse.java

```diff
@@ -0,0 +1,37 @@
+package com.woowacamp.soolsool.core.liquor.dto.response;
+
+import java.util.List;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+
+@Getter
+@RequiredArgsConstructor
+public class PageLiquorClickResponse {
+
+    private final boolean hasNext;
+    private final Long nextCursorId;
+    private final Long nextClickCount;
+    private final List<LiquorClickElementResponse> liquors;
+
+    public static PageLiquorClickResponse of(
+        final Pageable pageable,
+        final List<LiquorClickElementResponse> liquors
```

Comment:
```suggestion
        final List<Liquor> liquors
```

`PageLiquorResponse`처럼 Entity를 받으면 좋을 것 같아요.

Reply:
그거는 어렵습니다.. 저도 바꾸어봤는데, 마지막 클릭수를 가져올라면은 querydsl에서 liquor가 아닌 liquorClick이 합쳐진 dto를 받아야합니다. 쇼핑몰 미션 때도 cartProduct와product를 같이 하실때는 일급컬렉션 같이 정의해서 map해주었지만, 진우님께서, dto를 사용하는건 어떻냐고 해서 그렇게 했던 경험이 있습니다. entity를 반환해야 하는이유를 잘 모르겠습니다.

Reply:
지금 보니까 말씀하신 이유로 `LiquorQueryDslRepository.getListByClick()`가 Entity가 아닌 DTO를 반환하고 있네요. 하나의 DTO가 presentation layer, application layer, data layer 세 곳에서 사용하는데 괜찮은건가요?

Reply:
그럼dto를 새로 만드는게 날까요? 아니면 도메인을 하나 더 정의하는게 나을까요?

Reply:
별도의 비즈니스 로직이 없으니 Service-Repository 사이의 DTO가 적절할 것 같아요. 이런걸 해결하는 CQRS 패턴 같은게 있다고 하는데 자세히 공부는 안해봤어요

Reply:
dto 이용하면, 기존에 liquor ->liquorElementRespons를 바꾸어 주었는데 liquorClickElementResposne -> liquorElementResponse로 바꿀수있을거같아요